index.js

```diff
@@ -131,9 +179,19 @@ module.exports = options => {
     }
 
     const commandWithSemi = command + (command.endsWith(';') ? '' : ';')
+    const commandWithoutSemi = command.endsWith(';') ? command.substring(0, command.length - 1) : command
     return evalSol(commands.concat(commandWithSemi), options)
       .then(result => {
         commands.push(commandWithSemi)
+        if (Object.keys(specialCommands).indexOf(commandWithoutSemi) !== -1) {
+          let struct = {}
+          let results = result.toString().split(",")
```

Comment:
I believe `result` is always a string, so `.toString` is not needed. If it's not, we should make it clearer what the different types are and how they are handled.